lms/static/js/student_account/views/LoginView.js

```diff
@@ -163,7 +163,13 @@
                     successTitle = gettext('Check Your Email'),
                     successMessageHtml = HtmlUtils.interpolateHtml(
                         gettext(
-                          'Thank you! Please check your email for a message with password reset instructions.'), {
+                          '{paragraphStart}You entered {boldStart}{email}{boldEnd}. If this email address is ' +
+                          'associated with your {platform_name} account, we will send a message with password ' +
+                          'recovery instructions.{paragraphEnd}' +
+                          '{paragraphStart}If you do not receive a password reset message within 5 minutes, ' +
+                          'please verify that you have provided the correct email address, or check your spam ' +
+                          'folder. If you need further assistance, please {anchorStart}contact technical ' +
```

Comment:
Have you tested the link `contact technical support`? I think we should add link to a page which is helpful for the user or in other case as user can see the zendesk help widget on all pages we can change this to plain text so that their is no link which can cause issue or misleads the user.

Reply:
Yes, I talked to @ahmed-faraz46
Secondly, the link will work once we add a value for it via Site Configurations.

Reply:
We do not yet have a dedicated support url. But once we do, we can simply add it via admin later.